doc/user_manual/code_interfaces/accelerateCFD.tex

```diff
@@ -0,0 +1,178 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Okay, this will slow down my review a bit (since git, unlike svn has no understanding of file A turned into files B and C so I'll have to do a manual diff (well I'll use ediff)), but that works.